core/domain/feedback_domain.py

```diff
@@ -37,16 +37,20 @@ class FeedbackThread(python_utils.OBJECT):
         summary: str. A summary of the feedback thread.
         has_suggestion: bool. Whether the feedback thread includes a
             suggestion.
+        message_count: int. The number of messages posted onto the thread.
         created_on: datetime.datetime. The time when the feedback thread was
             created.
         last_updated: datetime.datetime. The time when the feedback thread
             was last updated.
+        last_message_text: str. The text of the last message in the thread.
+        last_message_author: str. The author of the last message in the thread.
```

Comment:
The thread domain object should store the user ID. This should be converted to a username only at the controller layer. Otherwise it's not resilient to possible future changes in author usernames.

Reply:
Done.